NGitLab.Tests/MembersClientTests.cs

```diff
@@ -1,14 +1,38 @@
-﻿using System.Globalization;
+﻿using System;
+using System.Globalization;
 using System.Linq;
 using System.Threading.Tasks;
 using NGitLab.Models;
 using NGitLab.Tests.Docker;
 using NUnit.Framework;
+#nullable enable
 
 namespace NGitLab.Tests
 {
     public class MembersClientTests
     {
+        [Test]
+        [NGitLabRetry]
+        public async Task AddMemberToProject()
+        {
+            using var context = await GitLabTestContext.CreateAsync();
+            var project = context.CreateProject();
+            context.CreateNewUser(out var user);
+            var projectId = project.Id.ToString(CultureInfo.InvariantCulture);
+
+            var expiresAt = DateTime.UtcNow.AddDays(30).ToString("yyyy-MM-dd");
```

Comment:
Use `DateTimeOffset` instead of `DateTime` (refer to this explanation of what the differences are: https://docs.microsoft.com/en-us/dotnet/standard/datetime/choosing-between-datetime).